nav2-usability/scenario_code/lib/deeplink-pathparam/deeplink_pathparam_auto_route.dart

```diff
@@ -65,7 +65,7 @@ class _BooksListScreenState extends State<BooksListScreen> {
               title: Text(book.title),
               subtitle: Text(book.author),
               onTap: () =>
-                  context.router.replace(BookDetailsRoute(id: books.indexOf(book))),
+                  context.router.pushNamed("/book/${books.indexOf(book)}"),
```

Comment:
If user deeplink to this page, will they see the backbutton?

Reply:
Yes I think we would like this to display an in-app back button to match the other samples

Reply:
Whoops, my bad - just added it